core/server/data/importer/importers/data/base.js

```diff
@@ -19,6 +19,19 @@ class Base {
             returnDuplicates: true
         };
 
+        this.legacyKeys = {};
+        this.legacyMapper = function legacyMapper(item) {
+            item = _.mapKeys(item, function matchLegacyKey(value, key) {
+                return self.legacyKeys[key] || key;
```

Comment:
Actually this is not comparable. settings : {key: 'activeApps'} would be mapping the value not the key. Had a little look to see if we could do something similar but it quickly got out of hand so keeping the settings key-value mapping separate is preferable